### PR TITLE
cmd/snap: `--last` for abort and watch, and aliases (search→find, change→tasks)

### PR DESCRIPTION
**Pull request by chipaca**

### CODE REVIEW

**Comment by chipaca** [comment]
The search alias is a fix for [https://bugs.launchpad.net/snappy/+bug/1697880](lp:1697880).

File: cmd/snap/cmd_abort.go [hunk 1]
```diff
@@ -25,11 +25,7 @@ import (
 	"github.com/snapcore/snapd/i18n"
 )
 
-type cmdAbort struct {
-	Positional struct {
-		ID changeID
-	} `positional-args:"yes" required:"yes"`
-}
+type cmdAbort struct{ lastMixin }
```
Comment (mvo5) [inline cmd/snap/cmd_abort.go]: Nice refactor! I wonder if there is a more fitting name for `lastMixin` - it is also capturing the `changeID` so to me it was not obvious at first that lastMixin also has this job. Trouble is I don't have a good suggestion either: `lastChangeMixin`, `changeIDmixin` also sounds not brilliant.

Comment (chipaca) [inline cmd/snap/cmd_abort.go]: To keep things interesting, I'll mention that last is also a noun.

the PR description is a bit cryptic, but the commits are nicely separate I think?

Comment (pedronis) [inline cmd/snap/cmd_abort.go]: I agree that changeIDMixin might be a better name

File: cmd/snap/last.go [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+var lastMixinOptDesc = map[string]string{
+	"last": i18n.G("Select last change of given type (install, refresh, remove, try, auto-refresh etc.)"),
+}
+
+var lastMixinArgDesc = []argDesc{{
+	name: i18n.G("<change-id>"),
+	desc: i18n.G("Change ID"),
+}}
+
+func (l *lastMixin) GetChangeID(cli *client.Client) (changeID, error) {
```
Comment (pedronis) [inline cmd/snap/last.go]: seems it would need less casts if this returned string and showChange took also a string

**Pull request merged**